### PR TITLE
Copy overview texts from previous mellon use case document

### PR DESCRIPTION
**Pull request by phochste**
- Copy 10000 view texts from mellon use cases document
- Removing double overview of all documents in all specs

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+index.html
```
Comment (mielvds) [inline .gitignore]: @phochste this prevents the build html from being accessble. I made this mistake before ;)

File: index.bs [hunk 2]
```diff
@@ -2,7 +2,7 @@
 Title: Overview of the ResearcherPod specifications
 Shortname: overview
 Level: 1
-Status: LS
+Status: DREAM
```
Comment (mielvds) [inline index.bs]: Can we at least make it a Living Document; these serve as project deliverables ;)

File: index.bs [hunk 3]
```diff
@@ -135,9 +96,148 @@ Of course, you are invited to [contribute]() any feedback, comments, or question
...
+
+Issue: Are the Rules and Inbox necesearily part of the Data Pod or can they exist as separate resources?
+
+The second network demonstrates a more classic setup with a browser Dashboard controlled by a server App which uses a
+Data Pod as backend storage.
+
+<div><img src="images/mellon-crud.svg" width="80%"></div>
+<i>CRUD operations in case the Dashoard and App is a classic client/server application</i>
+
+Communication between Data Pod and Scholarly Community network
```
Comment (mielvds) [inline index.bs]: I suggest that I and @JWerbrouck add a similar summary section about our use cases.

File: .gitignore (hunk 1 above)
Comment (phochste) [reply .gitignore]: ai ai ai

File: index.bs (hunk 2 above)
Comment (phochste) [reply index.bs]: Ok!

**Pull request merged**